functions.sh

```diff
@@ -387,7 +387,7 @@ backupCurrent ()
             mkdir -p "$rtlwifi_backup_dir"
             tar -czf "$rtlwifi_backup_outfile" -C "$rtlwifi_orig" . > /dev/null 2>&1
         else
-            echo "Could not backup rtlwifi because it could not be found!  Expected at $rtlwifi_orig" >&2
+            echo "Could not backup rtlwifi because it could not be found!  Expected at $rtlwifi_orig"
```

Comment:
Why remove the redirection to stderr?  Did this cause a problem or was it an accidental commit?

Reply:
sorry, accidental commit.